src/aap_eda/wsapi/consumers.py

```diff
@@ -111,14 +112,15 @@ async def handle_workers(self, message: WorkerMessage):
             )
             await self.send(text_data=extra_var_message.json())
 
-        controller_info = ControllerInfo(
-            url=settings.EDA_CONTROLLER_URL,
-            token=await self.get_awx_token(message),
-            ssl_verify=settings.EDA_CONTROLLER_SSL_VERIFY,
-        )
-
         await self.send(text_data=rulebook_message.json())
-        await self.send(text_data=controller_info.json())
+        awx_token = await self.get_awx_token(message)
+        if awx_token:
+            controller_info = ControllerInfo(
```

Comment:
@Alex-Izquierdo We would have test the removal of ControllerInfo with ansible-rulebook so it doesn't fail. It has [guards](https://github.com/ansible/ansible-rulebook/blob/c72a42087508e2aa03c0a183cf140dcc2b5aa093/ansible_rulebook/websocket.py#L76) for it but would be nice to know that existing DE's would keep working.

Reply:
@mkanoor In the migration we populate the activations with the existing token, activations without token works fine. I think there are no more cases.